### DISCUSSION TITLE
Cannot get Loupedeck Live to recognise v2.4.0 or v3.0.1 programming

### DISCUSSION THREAD

**Discussion by Aaron-C-Reskew** [calibx/msfsdeck#256] [Q&A] [answered]
Loupedeck Live installed without error. Windows Default Profile works.
When installing CaliBx MSFS plugin 2.4.0 or 3.0.1 they install with no problem.
When the preprogrammed keys on every page are mouse-overed in the UI "Action Not Available" shows.
None of the programming works on the unit unless I programme them manually.
I do not have any programming skills so cannot programme anything but the most basic of commands.

I have noticed just before I close down my desktop I get the following LogiPluginSvcExt.exe error which reads "The application was unable to start correctly [0xc0000142]. Click OK to close the application".
When researching the error code I carried out all the recommendations on the YouTube video https://www.youtube.com/watch?v=byyRbi_gaTs to no avail.

I read somewhere that in my Device Manager I should have "Remote NDIS Compatible Device" under Network Adaptors. This driver is not there and I cannot manually install it due to not having IP-Over-USB in the same location to enable it.
 
I ensured on each install and reinstall to switch off My Windows Defender and Virus Scanner in case these were a barrier to the plugins working.

OS: Windows 10 Pro

Is anyone able to solve this problem please.

**Comment by calibx** [comment]
Hi,

Can you verify the Loupedek UI version ? The plugin is not compatible with the 6.X release.

**Comment by Aaron-C-Reskew** [comment]
That will explain it. Why didn't Loupdeck Support inform me when I reported it to them twice (I'm still waiting for a reply after 2 weeks!) They even sent me your plugin.
I tried both v6.0.0.20753 and 6.0.0.20873.

Please advise which version will work and whether I will have to go back to Loupdeck Support. or are their links available.

**Reply by NJTravelerDH** [reply]
I have a copy of Loupedeck 5.9.  I uninstalled 6.x and installed 5.9.  My Loupedeck Live now works, giving a variety of values from MSFS 2020.  Dave (NJTraveler)

**Reply by calibx** [reply, marked as answer]
@Aaron-C-Reskew  : You have to stay on the 5.x version, the 5.9 is the last working one

**Reply by Aaron-C-Reskew** [reply]
Thank you to NJTraveler & CaliBx. You have no idea just how many hours I've spent on finding a solution this past month.
